Sources/Basic/StringConversions.swift

```diff
@@ -73,4 +73,212 @@ public extension String {
     public mutating func shellEscape() {
         self = shellEscaped()
     }
+    
+    /// Creates a string that contains only value C99 characters.
+    /// FIXME: We will likely want a better name...
+    public func mangledForC99() -> String {
```

Comment:
I think this should probably be named to follow "C99ExtendedIdentifier" which is the technical standard it follows.